test/test-versions.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 const path = require('path');
-const {browsers} = require('..');
+const browsers = require('..').browsers;
```

Comment:
This also causes a syntax error in non&#x2011;browser files to cause the tests to fail with an unhelpful error message (see&nbsp;https://github.com/mdn/browser-compat-data/issues/2330),

I'll look into fixing this in&nbsp;another&nbsp;PR.